docs/platforms/javascript/common/best-practices/multiple-sentry-instances.mdx

```diff
@@ -53,11 +55,23 @@ import {
   Scope,
 } from "@sentry/browser";
 
+// filter integrations that use the global variable
+const integrations = getDefaultIntegrations({}).filter(
+  (defaultIntegration) => {
+    return ![
+      "BrowserApiErrors",
+      "TryCatch",
+      "Breadcrumbs",
+      "GlobalHandlers",
+    ].includes(defaultIntegration.name);
+  },
+);
+
 const client = new BrowserClient({
   dsn: "___PUBLIC_DSN___",
   transport: makeFetchTransport,
   stackParser: defaultStackParser,
-  integrations: defaultIntegrations,
+  integrations: integrations,
```

Comment:
Maybe you want to write it this way, but I leave it up to you
```suggestion
  integrations,
```

Reply:
I thought about this, but this makes it especially explicit we are passing in this variable here. I'll leave this syntax for people who are less familiar with JS.